src/hrflow_connectors/connectors/poleemploi/warehouse.py

```diff
@@ -0,0 +1,202 @@
+    lessthanthree = "2"  # 2 -> De 1 à 3 ans d'expérience
+    morethanthree = "3"  # 3 -> Plus de 3 ans d'expérience
+
+
+class Qualification(IntEnum):
+    nonCadre = 0
+    cadre = 9
+
+
+class PeriodeSalaire(str, Enum):
+    Mensuel = "M"
+    Annuel = "A"
+    Horaire = "H"
+    Cachet = "C"
+
+
+class GrandDomaine(str, Enum):
+    agriculture = (  # A -> Agriculture/Pêche/
+        # /Espaces verts et naturels/Soins aux animaux
+        "A"
+    )
+    arts = "B"  # B -> Arts / Artisanat d’art
+    banque = "C"  # C -> Banque / Assurance
+    immobilier = "C15"  # C15 -> Immobilier
+    commerce = "D"  # D -> Commerce / Vente
+    communication = "E"  # E -> Communication / Multimédia
+    batiment = "F"  # F -> Bâtiment / Travaux Publics
+    hotellerie = "G"  # G -> Hôtellerie – Restauration / Tourisme / Animation
+    industrie = "H"  # H -> Industrie
+    installation = "I"  # I -> Installation / Maintenance
+    sante = "J"  # J -> Santé
+    services = "K"  # K -> Services à la personne / à la collectivité
+    spectacle = "L"  # L -> Spectacle
+    sport = "L14"  # L14 -> Sport
+    achats = "M"  # M -> Achats / Comptabilité / Gestion
+    direction = "M13"  # M13 -> Direction d’entreprise
+    conseil = "M14"  # M14 -> Conseil/Etudes
+    ressources = "M15"  # M15 -> Ressources Humaines
+    secretariat = "M16"  # M16 -> Secrétariat/Assistanat
+    marketing = "M17"  # M17 -> Marketing /Stratégie commerciale
+    informatique = "M18"  # M18 -> Informatique / Télécommunication
+    transport = "N"  # N -> Transport / Logistique
+
+
+class PublieeDepuis(IntEnum):
+    # Valeurs possibles : 1, 3, 7, 14, 31
+    a = 1
+    b = 3
+    c = 7
+    d = 14
+    e = 31
+
+
+class ModeSelectionPartenaires(str, Enum):
+    inclus = "INCLUS"
+    exclu = "EXCLU"
+
+
+class DureeHebdo(str, Enum):
+    nonPrecise = "0"
+    tempsPlein = "1"
+    tempsPartiel = "2"
+
+
+class ReadJobsParameters(BaseModel):
+    access_token: str = Field(
+        ..., description="Bearer Token used to access Pole Emploi's API", repr=False
+    )
+    range: t.Optional[str]
+    sort: t.Optional[int]
+    # domaine : t.Optional[Domaine]
+    # codeROME : t.Optional[codeROME]
+    # theme : t.Optional[Theme]
+    # appellation : t.Optional[Appellation]
+    # secteurActivite : t.Optional[SecteurActivite]
+    experience: t.Optional[Experience]
+    # typeContrat : t.Optional[TypeContrat]
+    # natureContrat : t.Optional[NatureContrat]
+    origineOffre: t.Optional[OrigineOffreTag]
+    qualification: t.Optional[Qualification]
+    tempsPlein: t.Optional[bool]
+    # commune : t.Optional[Commune]
+    distance = (
+        10  # distance=0 pour pour obtenir seulement les offres d'une commune spécifique
+    )
+    departement: t.Optional[str]
+    inclureLimitrophes: t.Optional[bool]
+    # region : t.Optional[Region]
+    # paysContinent : t.Optional[PaysContinent]
+    # niveauFormation : t.Optional[NiveauFormation]
+    # permis : t.Optional[Permis]
```

Comment:
@nedhirouebnou The commented lines group them and put them at the end. Try to add a comment explaining the need to get the list of values from an endpoint etc ... be explicit, so the next person working on your code can understand why you left them commented, or even you in a few months you won't be able to remember why you did this 
ZEN of PYTHON : "Explicit is better than Implicit"

Reply:
These are the fields for which I implemented the references in the later commit.